data/mods/DinoMod/monsters/zed-dinosaur.json

```diff
@@ -835,5 +835,25 @@
     "armor_bullet": 1,
     "description": "A small four-legged plant-eating dinosaur juvenile corpse with natural bone armor plates.",
     "upgrades": { "half_life": 14, "into": "mon_zalamosaurus" }
+  },
+  {
+    "type": "MONSTER",
+    "id": "mon_zianzhousaurus",
+    "name": { "str": "Qianz zombie" },
+    "looks_like": "mon_zyrannosaurus",
+    "volume": "550 L",
+    "weight": "770 kg",
+    "speed": 175,
+    "melee_skill": 10,
+    "melee_dice": 6,
+    "melee_dice_sides": 7,
+    "melee_cut": 6,
+    "armor_bash": 10,
+    "armor_cut": 10,
+    "armor_bullet": 20,
+    "hp": 230,
+    "description": "An undead Qianzhousaurus, looking at you with a furious rage in its eyes and seeming intent on eating you.",
+    "upgrades": { "half_life": 14, "into_group": "GROUP_Zianzhousaurus_UPGRADE" },
```

Comment:
```suggestion
```
Either cut this or define this upgrade group and create at least two upgrades for it.